pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -0,0 +1,173 @@
+package ipfailover
+
+import (
+	"fmt"
+	"io"
+
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/openshift/origin/pkg/cmd/util/variable"
+	ipf "github.com/openshift/origin/pkg/ipfailover"
+	"github.com/openshift/origin/pkg/ipfailover/keepalived"
+)
+
+const shortDesc = "Configure or view IP Failover configuration"
+const description = `
+Configure or view IP Failover configuration
+
+This command helps to setup IP Failover configuration for an OpenShift
+environment. An administrator can configure IP failover on an entire
+cluster or as would normally be the case on a subset of nodes (as defined
+via a labelled selector).
+
+If an IP failover configuration does not exist with the given name,
+the --create flag can be passed to create a deployment configuration and
+service that will provide IP failover capability. If you are running in
```

Comment:
You don't need a service - or at least, I can't think of a reason you would need one.